Form1.cs

```diff
@@ -73,103 +73,43 @@ public void TagsLostFocus(object sender, EventArgs e)
 

```

Comment:
Remove excess code and simplify, since from my eyes, the code on either side of the if statement were exactly the same.